README.md

```diff
@@ -270,3 +284,84 @@ Example:
     </tr>
 
 </table>
+
+
+## Sample usage in a NextJs project
+
+1. Make a component named Cookie.tsx in your components folder with the following content.
+```tsx
+'use client';
+
+import { CookieBanner } from '@schlomoh/react-cookieconsent';
+import React from 'react';
+
+const MyCookieBanner = () => {
+    const Content = () => (
+        <>
+            <h3>Life is better with cookies 🍪</h3>
+            <p>This website uses cookies to improve your experience. We’ll assume you’re ok with this, but you can opt out if you wish. <a href="/cookie" className='text-blue underline'>Cookie Policy</a></p>
+        </>
+    )
+
+    const MContent = () => (
+        <>
+            <h3>Manage your cookie settings.</h3>
+            <p>We use cookies and other tracking technologies to improve your experience on our website. We may store and/or access information on a device</p>
+        </>
```

Comment:
I feel like using only lorem ipsum here is safer... after all this might be interpreted as legal advice 😅

Reply:
done @Schlomoh 

Also, when using the component, whenever I hit refresh, the cookie banner comes and goes swifty (as if it glitches). Maybe you can improve it so that after accepting or rejecting, it does not load and show glitches.

Reply:
Let's create another issue for that 👍🏼